Maple2.Server.Game/Commands/BuffCommand.cs

```diff
@@ -46,17 +46,18 @@ private void Handle(InvocationContext ctx, int buffId, int level, int stack, int
             }
 
             if (duration > 0) {
-                duration *= 1000; // convert to milliseconds
+                duration = (int) TimeSpan.FromMilliseconds(1).TotalMilliseconds * duration;
```

Comment:
this makes no sense at all

Reply:
its the same just bit more explicit on what's being multiplies

Reply:
oops...its supposed to be FromSeconds

Reply:
TimeSpan.FromMilliseconds(1).TotalMilliseconds = 1 ms tho